modmesh/app/euler1d.py

```diff
@@ -104,41 +104,77 @@ def update(self, adata, ndata):
         self.num.figure.canvas.draw()
 
 
-class SolverConfig():
+class _Accessor:
```

Comment:
Every class (added/changed in the PR) should follow the Python 2/3 compatible declaration (inherit from `object`).

Reply:
Done!